src/index.html

```diff
@@ -1,18 +1,82 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
```

Comment:
The `crossorigin` attribute value should be quoted as `"anonymous"` to maintain consistency and adhere to the code style rules.